<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoList</class>
 <widget class="QWidget" name="TodoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="ListFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background-color: rgb(31,31,31);
border-bottom: 2px solid rgb(220,220,220);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="CheckBox">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NameLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: rgb(220,220,220)
}</string>
       </property>
       <property name="text">
        <string>Todo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="StartLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: rgb(220,220,220)
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EndLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: rgb(220,220,220)
}</string>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(220,220,220);
border: none;
}</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DelBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>39</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(220,220,220);
border: none;
}</string>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
